app/models/katello/rpm.rb

```diff
@@ -0,0 +1,65 @@
+module Katello
+  class Rpm < Katello::Model
+    include Glue::Pulp::PulpContentUnit
+    include Glue::Pulp::Rpm
+
+    PULP_SELECT_FIELDS = %w(name epoch version release arch checksumtype checksum)
+    PULP_INDEXED_FIELDS = %w(name version release arch epoch sourcerpm checksum filename _id)
+    CONTENT_TYPE = "rpm"
+
+    has_many :repositories, :through => :repository_rpms, :class_name => "Katello::Repository"
+    has_many :repository_rpms, :class_name => "Katello::RepositoryRpm", :dependent => :destroy, :inverse_of => :rpm
+
+    scoped_search :on => :name, :complete_value => true
+    scoped_search :on => :version, :complete_value => true
+    scoped_search :on => :release, :complete_value => true
+    scoped_search :on => :arch, :complete_value => true
+    scoped_search :on => :epoch, :complete_value => true
+    scoped_search :on => :filename, :complete_value => true
+    scoped_search :on => :sourcerpm, :complete_value => true
+    scoped_search :on => :checksum
+
+    before_save lambda { |rpm| rpm.summary= rpm.summary.truncate(255) unless rpm.summary.blank? }
```

Comment:
Should this be `rpm.summary =` (note the missing blank) ? Might explain why I was seeing no summaries?